prow/plugins/plugins.go

```diff
@@ -514,6 +514,25 @@ func (pa *PluginAgent) Config() *Configuration {
 	return pa.configuration
 }
 
+// ensureConfig ensure that the configuration
+// doesn't contain any nil value
+func ensureConfig(config *Configuration) error {
+	var errors []string
+	for _, configuration := range config.Plugins {
+		for _, plugin := range configuration {
+			if plugin == "heart" && config.Heart == nil {
+				errors = append(errors, "nil struct: heart")
```

Comment:
Could use a more verbose error that explained which org/repo the plugin was enabled for, and that the config for it was `nil`

Reply:
Alright im working on it !